### ISSUE TITLE
Local analysis with Firefox

### ISSUE THREAD

**Issue by gbtami** [gbtami/pychess-variants#327]
WASMX is working now with Firefox on lichess.org.
It works Ok on pychess.org with Chrome/Chromium and with Firefox nightly also.
(For latter you have to enable dom.postMessage.sharedArrayBuffer.bypassCOOP_COEP.insecure.enabled
in about:config)
But.
If I want to let it work with latest Firefox and add https://github.com/gbtami/pychess-variants/pull/326 (See requirements in https://github.com/niklasf/stockfish.wasm) local analysis doesn't work nor with Chrome nor with Firefox.

**Comment by notruck** [comment]
I found this in the Firefox documentation. 
https://developer.mozilla.org/en-US/docs/Glossary/same-origin_policy 

Lichess is able to serve the WASMX binary from the `same-origin` as defined above, so they don't need to enable the insecure bypasses required by Firefox Nightly. The reason why Chrome is working is probably because we enabled the experimental wasm flags manually. Lichess users do not have to do that.

![2020-09-03-110901_1010x610_scrot](https://user-images.githubusercontent.com/56622488/92151658-5ef32a00-edd6-11ea-8863-fff79fc1f063.png)

Also of possible interest is the following warning/note [at the top of this page:](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cross-Origin-Resource-Policy) 
![2020-09-03-111129_931x225_scrot](https://user-images.githubusercontent.com/56622488/92151808-8fd35f00-edd6-11ea-833c-8d2d51011304.png)
